script/testing/test

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-cd $(dirname $(dirname  $(dirname $0)))/controllers
+export PYTHONPATH=$(realpath ./stubs):$(realpath ./modules)
 
-export PYTHONPATH=$(realpath ../stub-webots)
-
-python3 -m unittest discover --start-directory sr_controller "$@"
+python3 -m unittest discover --start-directory modules "$@"
```

Comment:
Since we're no longer `cd`ing here, `modules` is, I think, an unreliable name.

I suggest sticking with the `cd` approach, even if the paths are updated.

Reply:
Not sure I see the difference between `cd`ing into the directory vs pointing `unittest` at it?

Reply:
058331824b18530fd728cc7571eba70478755aae